app/parse.py

```diff
@@ -15,8 +22,122 @@ class Product:
     num_of_reviews: int
 
 
+def init_driver() -> webdriver.Chrome:
+    options = webdriver.ChromeOptions()
+    options.add_argument("--headless")
+    options.add_argument("--no-sandbox")
+    options.add_argument("--disable-dev-shm-usage")
+    driver = webdriver.Chrome(options=options)
+    return driver
+
+
+def accept_cookies(driver: webdriver.Chrome) -> None:
+    try:
+        cookies_button = WebDriverWait(driver, 10).until(
+            ec.element_to_be_clickable(
+                (By.CLASS_NAME, "accept-cookies-button")
+            )
+        )
+        cookies_button.click()
+    except Exception:
+        pass
```

Comment:
Catching a general `Exception` without handling it or logging the error can make debugging difficult. Consider logging the exception to understand what went wrong if the cookies button is not found.